kubernetes_platform/python/docs/conf.py

```diff
@@ -0,0 +1,262 @@
+    'repo_name':
+        'pipelines',
+    'repo_type':
+        'github',
+    'edit_uri':
+        'blob/master/docs',
+    'globaltoc_collapse':
+        False,
+    'features': [
+        'navigation.expand',
+        # "navigation.tabs",
+        # "toc.integrate",
+        'navigation.sections',
+        # "navigation.instant",
+        # "header.autohide",
+        'navigation.top',
+        # "navigation.tracking",
+        'search.highlight',
+        'search.share',
+        'toc.follow',
+        'toc.sticky',
+    ],
+    'palette': [
+        {
+            'media': '(prefers-color-scheme: dark)',
+            'scheme': 'slate',
+            'primary': 'kfpblue',
+            # "accent": "lime",
+            'toggle': {
+                'icon': 'material/lightbulb',
+                'name': 'Switch to light mode',
+            },
+        },
+        {
+            'media': '(prefers-color-scheme: light)',
+            'scheme': 'default',
+            'primary': 'kfpblue',
+            # "accent": "light-blue",
+            'toggle': {
+                'icon': 'material/lightbulb-outline',
+                'name': 'Switch to dark mode',
+            },
+        },
+    ],
+    'font': {
+        'text': 'Open Sans'
+    },
+    'version_dropdown':
+        True,
+    'version_info': [{
+        'version':
+            'https://kfp-kubernetes.readthedocs.io/en/kfp-kubernetes-0.0.1/',
+        'title':
+            '0.0.1',
+        'aliases': ['stable'],
+    },],
+    # "toc_title_is_page_title": True,
+}
+# Add any paths that contain templates here, relative to this directory.
+templates_path = ['_templates']
+
+# The suffix(es) of source filenames.
+# You can specify multiple suffix as a list of string:
+source_suffix = '.rst'
+
+# The master toctree document.
+master_doc = 'index'
+
+# The language for content autogenerated by Sphinx. Refer to documentation
+# for a list of supported languages.
+#
+# This is also used if you do content translation via gettext catalogs.
+# Usually you set "language" from the command line for these cases.
+language = 'en'
+
+# List of patterns, relative to source directory, that match files and
+# directories to ignore when looking for source files.
+# This pattern also affects html_static_path and html_extra_path.
+exclude_patterns = ['_build', 'Thumbs.db', '.DS_Store']
+
+# The name of the Pygments (syntax highlighting) style to use.
+pygments_style = None
+
+# The default sidebars (for documents that don't match any pattern) are
+# defined by theme itself.  Builtin themes are using these templates by
+# default: ``['localtoc.html', 'relations.html', 'sourcelink.html',
+# 'searchbox.html']``.
+#
+# html_sidebars = {}
+
+# -- Options for HTMLHelp output ---------------------------------------------
+
+# Output file base name for HTML help builder.
+htmlhelp_basename = 'KfpKubernetesdoc'
+
+# -- Options for LaTeX output ------------------------------------------------
+
+latex_elements = {
```

Comment:
Can this be removed if we don't ever need latex?

Reply:
Yes, good thinking. Done.